docs/API/dynamic-content-api.md

```diff
@@ -30,6 +34,9 @@ There are 3 important parts of the API documentation that Flotiq provides for yo
 2. The descriptions of these endpoints contain all the information regarding the structure and constraints of the Content Type you defined.
 3. The API docs are enriched with code samples in several popular languages, which make it extremely easy to integrate your content into any external application.
 
+!!! note
+    The generated API documentation from the application provides your read-only key. If you wish to perform write actions, please enter your own API key.
```

Comment:
```suggestion
    The generated API documentation uses your read-only key to keep your account secure. If you wish to perform write actions, please enter your read-write API key.
```
The user has 2 keys. We need to tell the user which key to use. I also fixed the grammar a little.